plugins/discourse-presence/assets/javascripts/discourse/components/composer-presence-display.hbs

```diff
@@ -1,23 +1,30 @@
-{{#if this.shouldDisplay}}
-  <div class="presence-users">
-    <div class="presence-avatars">
-      {{#each this.presenceUsers as |user|}}
-        {{avatar user imageSize="small"}}
-      {{/each}}
-    </div>
-    <span class="presence-text">
-      <span class="description">
-        {{~#if this.isReply~}}
-          {{i18n "presence.replying" count=this.presenceUsers.length}}
-        {{~else~}}
-          {{i18n "presence.editing" count=this.presenceUsers.length}}
-        {{~/if~}}
-      </span>
-      <span class="wave">
-        <span class="dot">.</span>
-        <span class="dot">.</span>
-        <span class="dot">.</span>
+<div
+  {{did-insert this.setupChannels}}
+  {{did-update this.setupChannels @model.reply @model.whisper}}
```

Comment:
it might be better to create a modifier from scratch. if the modifier will reference `this.state` that will resolve the issue regarding component customization (mentioned in chat)